zooniverse/front-end-monorepo
# app-root, lib-user: Remove stats date constraint

## Package
- app-root
- lib-user

## Describe your changes
- remove stats date constraint (`'2015-03-17'`), was start of panoptes stats, not longer needed pending import of Oroboros stats
- refactor CertificateContainers per `user.created_at`
- ^ notably, refactor app-root CertificateContainer if no `start_date` param provided, set and update param per `user.created_at`
- remove Certificate Tip if `start_date` before date constraint, largely revert of #6191 

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What Zooniverse project should my reviewer use to review UX?
- What user actions should my reviewer step through to review this PR?
  - app-root
    - load your Certificate page without params, i.e. https://local.zooniverse.org:3000/users/markb-panoptes/stats/certificate
    - load your user stats page, select various date ranges, including 'All Time'
    - load your Certificate page with various date ranges
    - load a group stats page, select various date ranges, including 'All Time'
- Which storybook stories should be reviewed? n/a
- Are there plans for follow up PR’s to further fix this bug or develop this feature? no

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

## Refactoring
